test/before-after.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-env mocha */
+
+'use strict'
+
+const { bootTwoGrapes, killGrapes } = require('./helper')
+
+let grapes
+function setupHooks (cb) {
+  beforeEach(function (done) {
+    this.timeout(20000)
+    bootTwoGrapes((err, g) => {
+      if (err) throw err
+      grapes = g
+      grapes[0].once('announce', () => {
```

Comment:
`() => { done() }` could be written simply as `done`

Reply:
yes, i like the callback added, so its fast to hook in other stuff for testing or debugging and i don't need to add a new callback first